src/Components/Toast/Style.css

```diff
@@ -0,0 +1,3 @@
+.padding2vw {
```

Comment:
just use the built-in bootstrap `pt-2` or `pt-3` whichever one is the right amount